### PR TITLE
Add example and fixed linting issue

### PR DESCRIPTION
**Pull request by iswanj**
@jsdir I added example usage like **ladda button** [demo](http://lab.hakim.se/ladda/), Also fixed linting issue ( Updated eslint-config-airbnb and it's dependancies ) and react

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "react-scripts": "0.6.1"
+  },
+  "dependencies": {
```
Comment (jsdir) [inline example/package.json]: We'll need to add `ladda` here as well.

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+This is a runnable demo showing how to use `react-ladda`
+
+## Running Example
+
+**In the example directory, run:**
+```
+$ npm install
+$ npm start
+```
+**Open [http://localhost:3000](http://localhost:3000) to view it in the browser.**
```
Comment (jsdir) [inline example/README.md]: I think we can remove this since `npm start` will automatically open the browser.

File: example/src/demo.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/* Styles used specifically for the demo page */
+
+body {
+	background: #f5f5f5;
```
Comment (jsdir) [inline example/src/demo.css]: Use 2 spaces for indentation

File: example/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# `react-ladda` Example
```
Comment (jsdir) [inline example/README.md]: Let's link to this page from the main `README.md`.

**Comment by iswanj** [comment]
@jsdir Fixed requested changes

**Pull request merged**